blog/content/post/2019-04-18-USB-sniffing-with-OpenVizsla.md

```diff
@@ -0,0 +1,195 @@
+---
+title: USB Sniffing With OpenVizsla
+cover: /covers/sniffing_usb_cover.jpg
+author: lukasz.wcislo
+layout: post
+published: false
+date: 2019-04-18
+
+tags:
+  - USB
+  - USB sniffing
+  - OpenVizsla
+categories:
+  - Miscellaneous
+  - Security
+
+---
+
+### Background
+
+**"OpenVizsla is a project to design a device that will allow the capture and
+inspection of USB traffic that will help with the reverse engineering and
+debugging of proprietary USB devices, and will also be an invaluable tool for
+developers working with USB and especially those who are using USB in
+embedded designs."**
+
+This is the first sentence of a Kickstarter project which was funded in 2010.
+It was donated with over 80.000 USD (what was more then it was expected)
+(and what turned out to be less than really needed) in a month.
+A pair of enthusiasts (bushing and pytey) with their friends put a lot of effort
+to create small, cheap and open sourced USB sniffer.
+
+After over two years of struggle, while no working prototype was ready,
+people over the internet were really nervous. Many of them were calling
```

Comment:
```suggestion
people over the Internet were really nervous. Many of them were calling
```

@lwcislo please apply grammerly I believe it can reveal some typos and grammar issues.